### PR TITLE
Implement the Media List Page

### PR DESCRIPTION
**Pull request by kenzley**

### CODE REVIEW

File: src/models/media-file.js [hunk 1]
```diff
@@ -146,42 +143,6 @@ export default class MediaFile extends Model {
           type: DataTypes.TEXT,
           allowNull: false,
         },
-        citation_article_title: {
-          type: DataTypes.TEXT,
-          allowNull: false,
-        },
```
Comment (ChenXingguo) [inline src/models/media-file.js:152]: Just to double check, are we using media-files-x-bibliographic-reference.js to replace those values? Do we have any pipeline to dump those values?

**Review by ChenXingguo — COMMENTED** [review]
Hi Kenzley, could you please briefly explain if there will be any influences for removing some fields from the media file model? I do not see replacements for them in this commit so I am wondering where/how would those values go/be used.

**Comment by kenzley** [comment]
> Hi Kenzley, could you please briefly explain if there will be any influences for removing some fields from the media file model? I do not see replacements for them in this commit so I am wondering where/how would those values go/be used.

These columns are unused and I plan to remove them from the Morphobank server. The PR to delete them is in: https://github.com/tair/morphobank/pull/65/commits/7ccc2963039105a632f021ea39d58b1ee28e4f27

Should I include all database changes done in V3 to this repo as well?

**Comment by ChenXingguo** [comment]
> > Hi Kenzley, could you please briefly explain if there will be any influences for removing some fields from the media file model? I do not see replacements for them in this commit so I am wondering where/how would those values go/be used.
> 
> These columns are unused and I plan to remove them from the Morphobank server. The PR to delete them is in: [tair/morphobank@7ccc296](https://github.com/tair/morphobank/commit/7ccc2963039105a632f021ea39d58b1ee28e4f27)
> 
> Should I include all database changes done in V3 to this repo as well?

Yes, I believe sooner or later we do want to update the V4 models and database so that it is in sync with the V3 database schema.

**Review by ChenXingguo — APPROVED** [review]

**Comment by ChenXingguo** [comment]
Right now, I am connecting the V4 code to a copy of the V3 database, and I will need to test if this model change could still work with the old (un-updated) database schema. If not, maybe we need to create a migration to mitigate that.

**Comment by kenzley** [comment]
> Right now, I am connecting the V4 code to a copy of the V3 database, and I will need to test if this model change could still work with the old (un-updated) database schema. If not, maybe we need to create a migration to mitigate that.

Sounds good. There were many changes made to the V3 database schema so we will need to do that eventually. I'll merge this PR but let me know if you need help with the new database export.

**Pull request merged**